handler/src/test/java/io/netty/handler/ssl/SslHandlerTest.java

```diff
@@ -566,7 +572,7 @@ public void operationComplete(Future<Channel> future) {
         }
     }
 
-    @Test(timeout = 30000)
+    @Test(timeout = 60000)
```

Comment:
This test now does 4x more work than it does before and the time constraints have changed. The point of the timeout is to catch "hung" tests and not performance issues and so to avoid false positives the timeout was doubled.